crates/cairo-lint-core/tests/test_files/manual/manual_assert

```diff
@@ -0,0 +1,79 @@
+//! > check-if-then
```

Comment:
add more tests for example one with `if else` don't only check the happy path you need to think of edge cases